esmvaltool/recipes/recipe_climwip.yml

```
@@ -0,0 +1,122 @@
+# ESMValTool
+# recipe_climwip.yml
+---
+documentation:
+  description: EUCP ClimWIP
+
+  authors:
+    - kalverla_peter
+    - smeets_stef
+    - brunner_lukas
+    - camphuijsen_jaro
+
+  maintainer:
+    - kalverla_peter
+    - smeets_stef
+    - brunner_lukas
+
+  references:
+    - brunner2019
+    - lorenz2018
+    - knutti2017
+
+datasets: &model_data
+  - {dataset: ACCESS1-0, project: CMIP5, exp: [historical, rcp45], ensemble: r1i1p1}
+  - {dataset: ACCESS1-0, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+  - {dataset: ACCESS1-3, project: CMIP5, exp: [historical, rcp45], ensemble: r1i1p1}
+  - {dataset: ACCESS1-3, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+  - {dataset: CanESM2, project: CMIP5, exp: [historical, rcp45], ensemble: "r(1:5)i1p1"}
+
+  - {dataset: CCSM4, project: CMIP5, exp: [historical, rcp45], ensemble: "r(1:4)i1p1"}
+  - {dataset: CCSM4, project: CMIP5, exp: [historical, rcp60], ensemble: "r(1:4)i1p1"}
+  - {dataset: CCSM4, project: CMIP5, exp: [historical, rcp85], ensemble: "r(1:4)i1p1"}
+
+  - {dataset: CSIRO-Mk3-6-0, project: CMIP5, exp: [historical, rcp26], ensemble: "r(1:10)i1p1"}
+
+  - {dataset: BNU-ESM, project: CMIP5, exp: [historical, rcp26], ensemble: r1i1p1}
+  - {dataset: BNU-ESM, project: CMIP5, exp: [historical, rcp45], ensemble: r1i1p1}
+  - {dataset: BNU-ESM, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
```

```suggestion
  - {dataset: ACCESS1-0, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
  - {dataset: ACCESS1-3, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
  - {dataset: CCSM4, project: CMIP5, exp: [historical, rcp85], ensemble: "r(1:4)i1p1"}
  - {dataset: BNU-ESM, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
```

Emission paths should not be mixed. Focus only on RCP8.5 here.